server/src/internalClusterTest/java/org/opensearch/gateway/RecoveryFromGatewayIT.java

```diff
@@ -1366,7 +1366,8 @@ public void testSingleShardStoreFetchUsingBatchAction() throws ExecutionExceptio
         DiscoveryNode[] nodes = getDiscoveryNodes();
         TransportNodesListShardStoreMetadataBatch.NodesStoreFilesMetadataBatch response = prepareAndSendRequest(
             new String[] { indexName },
-            nodes
+            nodes,
+            false
```

Comment:
Could you explain why this case doesn't need the index to be green?

Reply:
I have tested that if we don't define the number of nodes in the cluster, there will be one data node by default. The created index has 1 replicas, so it will never be green.

Reply:
Thanks. Please merge the latest code of main branch and make sure the gradle checks can pass.